scripts/get-test-pass-rate.sh

```diff
@@ -0,0 +1,27 @@
+# This script is used in the "runAfterAll" stage in our E2E test pipeline. It's used 
+# to get timeline and metrics data so that the test pass rate can be calculated in a later step.
+
+echo BUILD_ID=$BUILD_ID
```

Comment:
Nit: since most devs on the team don't work with shell scripts often, I always recommend adding inline comments to explain the steps being taken - will help devs get up to speed on what's going on if they ever need to make changes.